core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -206,6 +206,10 @@ Queue.WaitingForNextAvailableExecutorOn=Waiting for next available executor on \
 Queue.init=Restoring the build queue
 Queue.node_has_been_removed_from_configuration={0} has been removed from configuration
 Queue.executor_slot_already_in_use=Executor slot already in use
+Queue.ExceptionCanTake=Exception evaluating if the node can take the task
+Queue.ExceptionCanTakeLog=Exception evaluating if the node ''{0}'' can take the task ''{1}''
+Queue.ExceptionCanRun=Exception evaluating if the queue can run the task
+Queue.ExceptionCanRunLog=Exception evaluating if the queue can run the task ''{0}''
```

Comment:
Log messages are generally not localized, I don't think we want to start it like this.

Trivial to roll back in a followup change though.